heron/ui/src/python/main.py

```diff
@@ -134,14 +136,15 @@ def main():
 
   # log additional information
   command_line_args = vars(parsed_args)
-  address = socket.gethostbyname(socket.gethostname())
+
+  address = '0.0.0.0' if command_line_args['address'] is None else command_line_args['address']
```

Comment:
or instead of the long line - 

```
address = socket.gethostbyname(socket.gethostname())
if command_line_args['address']:
   address = command_line_args['address']
```
easy to read as well, thoughts?